board/opentrons/ot2/post-image.sh

```diff
@@ -55,6 +55,11 @@ cat << __EOF__ >> "${BINARIES_DIR}/rpi-firmware/config.txt"
 dtparam=audio=on
 __EOF__
 
+dtc -@ -I dts -O dtb -o "${BINARIES_DIR}/rpi-firmware/overlays/gpio-revision-bits.dtbo" "${BOARD_DIR}/gpio-revision-bits.dts"
+cat << __EOF__ >> "${BINARIES_DIR}/rpi-firmware/config.txt"
+dtoverlay=gpio-revision-bits
```

Comment:
this will append `dtoverlay=gpio-revision-bits` to config.txt but that means there will be two `dtoverlay` entries, which might be problematic. We could instead either
- change this to use `sed` instead (see line 10 of this file)
- make a variable called like `overlays_to_add` or something and have the logic here and on lines 18-22 of this file add the names to it, and then have logic that emits all the names to config.txt